packages/common/src/fixtures/xform-attachments.ts

```diff
@@ -23,6 +23,9 @@ const xformAttachmentFileExtensions = [
 	'.xml',
 	'.xml.example',
 	'.xlsx',
+	'.png',
```

Comment:
@lognaturel I checked the Collect spec's Select with images, but didn't see the supported types for images. 
Is it expected gif and svg?

Reply:
I've added GIF and SVG extensions to this list

Reply:
Why not just let the browser support whatever it can? It's less precise than defining the set of supported image types but could offer some appreciated flexibility. I don't feel very strongly either way but philosophically I find it ok to be a little more loose for things like this, especially if it matches what Collect and Enketo do.

Reply:
[I left a comment about this here](https://github.com/getodk/web-forms/pull/399#discussion_r2085767466)